### PR TITLE
libgdata: 0.17.9 -> 0.17.10

### PR DESCRIPTION
**Pull request by worldofpeace**
###### Motivation for this change
* Meson!
* intiltool -> gettext
* multi-outputs
* installed tests

https://gitlab.gnome.org/GNOME/libgdata/blob/0.17.10/NEWS

Installed Tests are here but I didn't have time to actually add the NixOS test and have it working.
Additionally this had to go to staging because we needed Meson `>= 0.50.0`.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by 7c6f434c** [comment]
@worldofpeace you built it on NixOS with sandbox, right?

**Comment by worldofpeace** [comment]
> @worldofpeace you built it on NixOS with sandbox, right?

I don't think I had the time to actually build it :smile: 
I will do this now.

**Comment by worldofpeace** [comment]
Built `libgdata` and `gnome-photos` against 94e8afc254d7e4a419287513c4c2a2a833e8f511

File: pkgs/development/libraries/libgdata/installed-tests-path.patch [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+@@ -20,9 +20,9 @@ gdata_api_version_minor = 0
+ 
+ # Define the install directories
+ gdata_prefix = get_option('prefix')
+-gdata_datadir = get_option('datadir')
+-gdata_libexecdir = get_option('libexecdir')
+-gdata_includedir = get_option('includedir')
++gdata_datadir = gdata_prefix / get_option('datadir')
++gdata_libexecdir = gdata_prefix / get_option('libexecdir')
++gdata_includedir = gdata_prefix / get_option('includedir')
```
Comment (7c6f434c) [inline pkgs/development/libraries/libgdata/installed-tests-path.patch]: That seems to be a generic «multiple-outputs» thing, not test-related. Maybe rename the patch?

Comment (worldofpeace) [reply pkgs/development/libraries/libgdata/installed-tests-path.patch]: There's probably lots of other patches with the same name that do this.
I've actually been meaning to upstream several of them.

**Review by 7c6f434c — APPROVED** [review]

**Comment by worldofpeace** [comment]
Thanks @7c6f434c, let's go for it :confetti_ball:

**Pull request merged**